orquesta/utils/parameters.py

```diff
@@ -61,12 +64,16 @@ def parse_inline_params(s, preserve_order=True):
         v = re.sub("^'", '', v)
         v = re.sub("'$", '', v)
 
-        if v.lower() == 'true' or v.lower() == 'false':
-            v = v.lower()
+        curly_brace_in_param = bool(v[0] == '{' and v[len(v) -1] == '}')
```

Comment:
@jinpingh Can you provide some explanation why this curly_brace_in_param is needed?

Reply:
@jinpingh Ok. I read the description above. It's for JSON.  Do we have a unit test to cover JSON in single/double quotes?  If not, can you add?  Thanks.

Reply:
There are two test cases covered  with double quote e.g. `'x={"a": 1}'`.

Reply:
Can you add a comment above this line here to describe this check of curly_brace_in_param and what it is for?

Reply:
Also, can you add another test here for empty string (i.e. `x=""`)? I think the `v[len(v) -1]` is going to error.  Also, you can use `v[-1]` which is simpler.